nasa-gibs/worldview
# WV-2373: Adding Coordinate Format Selection to Settings Menu

## Description

Adding coordinate format settings button to the settings menu. This setting will update all of the coordinates on the screen including the coordinates in the place marker pin and the coordinates in the bottom right hand of the screen. 

## How To Test

Place a marker somewhere on the map. Go to the settings menu and change the coordinate format. You will see the coordinate format change on the marker and in the bottom right. You can also change the coordinate format with the bottom right coordinate format button and verify that the active selection in the settings menu has changed.
